sklearn/model_selection/_validation.py

```diff
@@ -482,13 +490,22 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     estimator : estimator object
         The fitted estimator
     """
+    progress_msg = ""
+    if verbose > 2:
+        if split_progress is not None:
+            progress_msg = f" {split_progress[0]+1}/{split_progress[1]}"
```

Comment:
You don't think this should say "split" in it?

Reply:
I was under the impression that it was too verbose (see [comment](https://github.com/scikit-learn/scikit-learn/pull/8281#issuecomment-311999516)). Perhaps I should reintroduce split and not candidate?

Reply:
A bit unclear, but that comment includes a line in the output providing a legend.

`[CV (fold; candidate)]`

Maybe it's fine without?

Reply:
But it was also clearer there where the parameter values were shown explicitly.

Reply:
You are right, I removed the legend because I felt as the line
```
Fitting 5 folds for each of 4 candidates, totalling 20 fits
```
was enough.
I see you are open to this solution, so if you don't mind I'm keeping it this way. Thanks for your comprehension.